tests/unit/models/project-test.js

```diff
@@ -465,11 +465,6 @@ describe('models/project.js', function() {
       var addon = project.findAddonByName('foo-ba');
       expect(addon).to.equal(undefined, 'foo-ba should not be found');
     })
-
-    it('should return an addon that is the unqualified name of requested name', function() {
```

Comment:
can you briefly describe why these are missing, i may be missing sufficient context